setup.py

```diff
@@ -57,8 +57,8 @@ def run_tests(self):
           'Topic :: Scientific/Engineering :: Visualization',
           'Topic :: Database :: Front-Ends',
           'Programming Language :: Python :: 3',
-          'Programming Language :: Python :: 3.5',
           'Programming Language :: Python :: 3.6',
           'Programming Language :: Python :: 3.7',
-          'Programming Language :: Python :: 3.8'],
+          'Programming Language :: Python :: 3.8',
+          'Programming Language :: Python :: 3.9'],
```

Comment:
Python 3.10 is out :)
```suggestion
          'Programming Language :: Python :: 3.9',
          'Programming Language :: Python :: 3.10'],
```